core/src/test/java/org/elasticsearch/bwcompat/ClusterStateBackwardsCompatIT.java

```diff
@@ -102,6 +100,7 @@ private NodesInfoResponse clusterNodes() {
 
     private TransportClient newTransportClient() {
         Settings settings = Settings.settingsBuilder().put("client.transport.ignore_cluster_name", true)
+                .put("path.home", PathUtils.get(".").toAbsolutePath())
```

Comment:
is `PathUtils.get(".").toAbsolutePath()` correct? I see that we do `.put("path.home", createTempDir().toAbsolutePath())` in a lot of existing tests....